### ISSUE TITLE
New Canvas Block Palette 

### ISSUE THREAD

**Issue by Shidoengie** [Anuken/Mindustry-Suggestions#4279]
### Describe what you would like changed, and why.

this is a reopening of a stale issue:
#4191 
TLDR Im suggesting replacing the current palette present in the Canvas block a new palette
The reason for this is that currently the palette present in the canvas block is unnecessarily limited limits the users creativity.

### Describe the changes you want to propose. Include possible alternatives.

i propose exchanging the current palette with this palette 
https://lospec.com/palette-list/dawnbringers-8-color
```
000000
55415f
646964
d77355
508cd7
64b964
e6c86e
dcf5ff
```
and if not possible as this may require permission from drawnbridger, (creator of the palette), there could be a community context for deciding the new palette.
or even you could just have a color picker and a limit of 8 colors which would fix the problem with the previous canvas block schematics that no longer have those colors and allow for even more creativity!

**Comment by Shidoengie** [comment]
Or you could just have a color picker and a limit of 8 colors which would fix the problem with the previous canvas block schematics that no longer have those colors and allow for even more creativity !
